lib/messagebird/conversation_client.rb

```diff
@@ -5,13 +5,25 @@
 
 module MessageBird
   class ConversationClient < HttpClient
-    ENDPOINT  = 'https://conversations.messagebird.com/v1/'
+    attr_reader :endpoint
 
+    @endpoint  = 'https://conversations.messagebird.com/v1/'
+    WHATSAPP_SANDBOX_ENDPOINT = 'https://whatsapp-sandbox.messagebird.com/v1/'
+
+    def initialize(access_key, features=[])
```

Comment:
Is the addition of this constructor backwards compatible?

Reply:
Yes, if the features parameter isn't provided it will default to an empty list.